src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/DataSetsControllerTests.cs

```diff
@@ -703,4 +707,479 @@ private async Task<HttpResponseMessage> GetVersion(Guid dataSetId, string dataSe
+                .ForIndex(2, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 3")
+                    ]
+                ))
+                .ForIndex(3, dsv => dsv.SetVersionNumber(2, 1))
+                .ForIndex(3, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 4")
+                    ]
+                ))
+                .GenerateList();
+
+            await TestApp.AddTestData<PublicDataDbContext>(context =>
+                context.DataSetVersions.AddRange(dataSetVersions));
+
+            var response = await GetMeta(dataSet.Id, "2.0");
+
+            var content = response.AssertOk<DataSetMetaViewModel>(useSystemJson: true);
+
+            Assert.NotNull(content);
+            Assert.Equal("filter 3", content.Filters.Single().Label);
+        }
+
+        [Fact]
+        public async Task VersionUnspecified_ReturnsLatestVersion()
+        {
+            DataSet dataSet = DataFixture
+                .DefaultDataSet()
+                .WithStatusPublished();
+
+            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSets.Add(dataSet));
+
+            var dataSetVersions = DataFixture
+                .DefaultDataSetVersion(
+                    filters: 1,
+                    indicators: 1,
+                    locations: 1,
+                    timePeriods: 3)
+                .WithStatusPublished()
+                .WithDataSetId(dataSet.Id)
+                .ForIndex(0, dsv => dsv.SetVersionNumber(1, 0))
+                .ForIndex(0, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 1")
+                    ]
+                ))
+                .ForIndex(1, dsv => dsv.SetVersionNumber(1, 1))
+                .ForIndex(1, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 2")
+                    ]
+                ))
+                .ForIndex(2, dsv => dsv.SetVersionNumber(2, 0))
+                .ForIndex(2, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 3")
+                    ]
+                ))
+                .ForIndex(3, dsv => dsv.SetVersionNumber(2, 1))
+                .ForIndex(3, dsv => dsv.SetFilterMetas(() =>
+                    [
+                        DataFixture
+                        .DefaultFilterMeta()
+                        .WithLabel("filter 4")
+                    ]
+                ))
```

Comment:
Similar thing to other comment. Can roll the `ForIndex` calls up

Reply:
done :)